images.yml

```diff
@@ -48,6 +49,10 @@ config:
       version: "1.8.0"
       description: "OpenJDK 8"
       major: 8
+    openjdk8:
```

Comment:
Shouldn't this be openjdk11 ?

Reply:
wonder how the generated files were created, but I guess not with fish-pepper ?

Reply:
I didn’t use fish-pepper, but manually edited them.

Reply:
At the moment this doesn't work as when fish-pepper is running it will also create openjdk11 images for alpine and jboss which are obviousl invalid. Let me fix that and the push to your branch.